ui/src/map/layers/alignment/badge-layer-utils.ts

```diff
@@ -72,7 +73,7 @@ function createBadgeFeatures(
 
                             const offsetDirection = badgeRotation.drawFromEnd ? -1 : 1;
 
-                            ctx.moveTo(x + (badgeWidth + 6) * offsetDirection, y);
+                            ctx.moveTo(x + (badgeWidth + 7) * offsetDirection, y);
```

Comment:
Tässä puhuttaneen pikseleistä joten erotus ei ole suuri? Tämän taikanumeron voisi ehkä tehdä vakioksi, koska ainakaan näin kun tämä tulee vastaan reviewissä, ei ole kauheasti tietoa siitä mitä tämän kuuluisi kuvastaa.